client/src/components/profile/ProfileRequestForm.js

```diff
@@ -27,28 +33,101 @@ const useStyles = makeStyles(() => ({
 
 function ProfileRequestForm() {
   const classes = useStyles();
+
+  // Get dispatch method and state from auth context
+  // const dispatch = useContext(AuthDispatchContext);
+  const { user } = useContext(AuthStateContext);
+
   // Today's date to be passed as the minimum and default value to the 'drop off' date picker input.
   const today = new Date();
   const todayFormatted = today.toISOString().split("T")[0];
 
-  const [dropOffDate, setDropOffDate] = useState(todayFormatted);
-  const [pickUpDate, setPickUpDate] = useState(todayFormatted);
+  const [startDate, setStartDate] = useState(todayFormatted);
+  const [endDate, setEndDate] = useState(todayFormatted);
+  const [startTime, setStartTime] = useState("09:00:00");
+  const [endTime, setEndTime] = useState("10:30:00");
+  const [requestFormData, setRequestFormData] = useState({
```

Comment:
Setting start and end date to empty strings means that if I just send the request immediately without changing anything the request fails

Reply:
I have set the defaults to:
- start: 30 minutes from now rounded up to the next hour.
- end: 2 hours after the default start date.